toolset/setup/linux/languages/rust.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+RETCODE=$(fw_exists $IROOT/rust.installed)
+[ ! "$RETCODE" == 0 ] || { . $IROOT/rust.installed; return 0; }
+
+fw_get https://static.rust-lang.org/dist/rust-1.0.0-x86_64-unknown-linux-gnu.tar.gz
+fw_untar rust-1.0.0-x86_64-unknown-linux-gnu.tar.gz
+(
+	cd rust-1.0.0-x86_64-unknown-linux-gnu
+	./install.sh --prefix=$IROOT/rust
+)
+
+echo "export LD_LIBRARY_PATH=\${LD_LIBRARY_PATH}:\$IROOT/rust/lib" >> $IROOT/rust.installed
```

Comment:
Should be a single `>` here - always create/override the file on the first instance.